support-frontend/app/utils/CheckoutValidationRules.scala

```diff
@@ -102,6 +112,7 @@ object CheckoutValidationRules {
       case _: Contribution => PaidProductValidation.passes(createSupportWorkersRequest)
       case _: GuardianAdLite => PaidProductValidation.passes(createSupportWorkersRequest)
     }) match {
+      case Invalid(message) if PreservableErrorMessages.isMessagePreservable(message) => Invalid(message)
```

Comment:
This converts to a `invalidPostCode` message to be picked up in `errorReasons.tsx` later